ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-new_release.yaml

```diff
@@ -17,29 +17,35 @@ images:
   - name: BASE_OS
     value: redhat
   - name: PROJECT_VERSION
-    value: 2023.3.0
+    value: 2024.4.0
   - name: ov_source_branch
-    value: releases/2023/3
+    value: new_release
   - name: ov_source_org
-    value: openvinotoolkit
+    value: opendatahub-io
   - name: ov_contrib_branch
-    value: releases/2023/3
+    value: new_release
   - name: ov_contrib_org
-    value: openvinotoolkit
+    value: opendatahub-io
   - name: ov_use_binary
     value: "0"
   - name: sentencepiece
     value: "1"
   - name: CHECK_COVERAGE
-    value: "1"
+    value: "0"
   - name: RUN_TESTS
     value: "1"
   - name: debug_bazel_flags
-    value: --strip=always
+    value: --strip=always --define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0 --//:distro=redhat
```

Comment:
@heyselbi, @rpancham and @Xaenalt these parameters will need to be moved when this branch will become "release"